packages/node_modules/@node-red/nodes/core/common/20-inject.html

```diff
@@ -189,7 +189,29 @@
             }
             return lab;
         },
-        label: function() {
+        label: function() {        
+            if (Array.isArray(this.props)) {
```

Comment:
Rather than completely rewriting the label logic, I am creating payload & topic variables. This allows re-use of the established logic.